objects/obj_switch_rooms/Step_0.gml

```diff
@@ -0,0 +1,8 @@
+/// @description switch rooms
+if (keyboard_check_pressed(vk_tab) || gamepad_button_check_pressed(0,gp_select)) {
+	if (room = room_sam_movement) {
```

Comment:
It look like this room name might not be right. There's no `room_sam_movement` in your branch?